bve/src/parse/mesh/instructions/mod.rs

```diff
@@ -1,5 +1,21 @@
-use crate::parse::mesh::{BlendMode, GlowAttenuationMode, MeshError, Span};
-use crate::parse::util;
+//! Underlying instructions behind the parsing of a mesh.
+//!
+//! You may use this library if you need to specifically edit, or use the exact instructions.
+//!
+//! Three important functions in this library which must be run in order:
+//!
+//! - [`create_instructions`] takes a `&str` and parses it to instructions using a custom serde routine.
+//! - [`post_process`] postprocesses away difficult to execute instructions. Must be called before execution of the
```

Comment:
Do you just drop the difficult to execute instructions? If not, tell me what happens to them

Reply:
That is discussed in the post_process documentation.

Reply:
Resolved